src/modules/feature/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Applications, Feature } from './types'
+
+export const fetchFeaturesDelay = 10000
+
+export const fetchFeatures = async (applications: Applications[]) => {
+  const promises = applications.map(a => fetchFeature(a))
+  const features = await Promise.all(promises)
+
+  return features.reduce((acc, feature) => {
+    acc[feature.application] = feature
+    return acc
+  }, {} as Record<Applications, Feature>)
+}
+
+const fetchFeature = async (application: Applications): Promise<Feature> => {
+  const response = await fetch(
+    `https://feature-flags.decentraland.org/${application}.json`
```

Comment:
just a question cos i dunno how this service works, but since `.org` is hardcoded here i suppose that there's a single feature-flag environment, no? How do they handle different flags for different envs?

Reply:
The ff service allows for determining which referer has the flag enabled or not, so if I want a flag to only be enabled for dev, I could add the urls of dev sites (*.io and/or *.zone)